### PR TITLE
fix: depositAddress audit finding 3

### PR DESCRIPTION
**Pull request by nknavkal**
## Pull Request Title Format

Please ensure your PR title follows the [Conventional Commit](https://www.conventionalcommits.org/) format:

```
<type>[optional scope]: <description>
```

### Allowed Types:

- `feat`: A new feature
- `fix`: A bug fix
- `docs`: Documentation only changes
- `style`: Changes that do not affect the meaning of the code
- `refactor`: A code change that neither fixes a bug nor adds a feature
- `test`: Adding missing tests or correcting existing tests
- `chore`: Changes to build process or auxiliary tools
- `perf`: A code change that improves performance
- `ci`: Changes to CI configuration files and scripts
- `build`: Changes that affect the build system or external dependencies
- `revert`: Reverts a previous commit

### Examples:

✅ `feat: add cross-chain intent validation`
✅ `fix: resolve memory leak in prove method`
✅ `docs: update API documentation for routes`
✅ `refactor(inbox): simplify refund logic`
✅ `test: add edge case coverage for token transfers`
✅ `chore: update dependencies to latest versions`

### Important Notes:

- The description should be lowercase and not end with a period
- Use present tense ("add" not "added" or "adds")
- Scopes are optional but helpful for organizing changes
- If you need to bypass this check temporarily, add the label `skip-pr-title-check`

---

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Test improvements
- [ ] Code refactoring
- [ ] Performance improvement
- [ ] CI/Build changes

### CODE REVIEW

File: contracts/deposit/DepositFactory_USDCTransfer_Solana.sol [hunk 1]
```diff
@@ -147,15 +150,13 @@ contract DepositFactory_USDCTransfer_Solana {
         bytes32 destinationAddress,
         address depositor
     ) external returns (address deployed) {
-        if (destinationAddress == bytes32(0)) revert InvalidDestinationAddress();
```
Comment (fish-sammy) [inline contracts/deposit/DepositFactory_USDCTransfer_Solana.sol:154]: Why the removal?

Comment (nknavkal) [reply contracts/deposit/DepositFactory_USDCTransfer_Solana.sol:154]: this validation is already performed in initialize in the basefactory

**Review by fish-sammy — APPROVED** [review]
